<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="LoginPage">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>170</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>172</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QFrame" name="formFrame">
                <property name="styleSheet">
                 <string notr="true">#formFrame{
border: 1px solid rgb(24, 103, 155);
border-radius: 5px;
}</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="spacing">
                    <number>20</number>
                   </property>
                   <property name="leftMargin">
                    <number>20</number>
                   </property>
                   <property name="topMargin">
                    <number>20</number>
                   </property>
                   <property name="rightMargin">
                    <number>20</number>
                   </property>
                   <property name="bottomMargin">
                    <number>20</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Username:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Password:</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <property name="spacing">
                    <number>20</number>
                   </property>
                   <property name="leftMargin">
                    <number>20</number>
                   </property>
                   <property name="topMargin">
                    <number>20</number>
                   </property>
                   <property name="rightMargin">
                    <number>20</number>
                   </property>
                   <property name="bottomMargin">
                    <number>20</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="userInput">
                     <property name="minimumSize">
                      <size>
                       <width>150</width>
                       <height>26</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">border: 1px solid rgb(24, 103, 155);
border-radius: 5px;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="passwordInput">
                     <property name="minimumSize">
                      <size>
                       <width>150</width>
                       <height>26</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">border: 1px solid rgb(24, 103, 155);
border-radius: 5px;</string>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Password</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="topMargin">
                 <number>20</number>
                </property>
                <property name="bottomMargin">
                 <number>20</number>
                </property>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="loginButton">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid rgb(24, 103, 155);
border-radius: 5px;
background-color: rgb(124, 203, 255);
color: white;
/*background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #f00, stop:1 #000);*/</string>
                  </property>
                  <property name="text">
                   <string>Вход</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>172</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>169</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DashbordPage">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="chart1">
            <layout class="QVBoxLayout" name="verticalLayout_8"/>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="chart2">
            <layout class="QVBoxLayout" name="verticalLayout_9"/>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QFrame" name="chart3">
            <layout class="QVBoxLayout" name="verticalLayout_10"/>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QFrame" name="chart4">
            <layout class="QVBoxLayout" name="verticalLayout_7"/>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="chart5">
            <layout class="QVBoxLayout" name="verticalLayout_11"/>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QFrame" name="chart6">
            <layout class="QVBoxLayout" name="verticalLayout_12"/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
